src/scripts/index.ts

```diff
@@ -162,6 +164,10 @@ window.addEventListener('message', (event) => {
       }
       break
     }
+    case HostCommand.SetDebugMode: {
+      debugMode = true
```

Comment:
@tommy9 what thoughts do you have on scenarios where `debugMode` can be set back to `false` ?
At the minute it seems that once set, it remains true for the rest of the emulator view's session

Reply:
The debugger disposes the emulator panel when the debug session is terminated, so it should not be possible to have this situation currently. However, that was just a simple option I took when setting up the debugger and there should be a cleaner way to exit the debugger but keep the emulator going. I'll make a new issue for this.